### PR TITLE
163975550 Added the office view file to view folder - v_create_office.py

### PR DESCRIPTION
**Pull request by orregumathijr**
### What does this PR do?
It contains the view endpoint for my offices 
-Url link for creating an office
-Url link for getting all offices
-Url link for to get a specific office
-Url link for Deleting an office

### Description of Task to be completed?
* An api view endpoint for creating a political office

### How should this be manually tested?
* Clone the repo by git clone : https://github.com/orregumathijr/app.git
* Navigate into the project when inside my virtual env
 *Run the project from the main file myapp.py

### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/163975550

### CODE REVIEW

**Comment by Nduhiu17** [comment]
You can change the response code for delete to 204. Then,let the next feature be on its own branch

File: v1/api/views/v_create_office.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
+from flask import make_response, jsonify, request, abort, Blueprint
+from app.api.v1.models.create_office import ModelOffice, offices
+# from utils.validations import raise_error, check_office_keys, on_success, office_restrictions
```
Comment (Quantum-35) [inline v1/api/views/v_create_office.py]: Remove the commented code.

**Review by Quantum-35 — CHANGES_REQUESTED** [review]

File: v1/api/views/v_create_office.py (hunk 1 above)
Comment (orregumathijr) [reply v1/api/views/v_create_office.py]: Done!

**Comment by Quantum-35** [comment]
Looks good. You can merge it.

**Pull request merged**